src/store/profileReducer.js

```diff
@@ -18,27 +18,34 @@ const initialState = {
 };
 
 const defaultLikesValue = 0;
+const nextId = 8;
```

Comment:
As far as I can see, now you've got the same "nextId" default constant in several files.
Consider creating a separate "core" folder where you'll have "constants.js" file containing all the constant values of your project.